lib/fine-types/test/Language/FineTypes/PackageSpec.hs

```diff
@@ -1,42 +1,167 @@
+{-# LANGUAGE OverloadedLists #-}
```

Comment:
😲 I did not know that this extension existed.

I see, it's used to avoid excessive use of `Map.fromList`, `Set.fromList` in the example. Fair enough, the extension is syntactic sugar and has little risk of breaking; happy to allow it in test code like this.